api/conftest.py

```diff
@@ -844,7 +845,11 @@ def github_repository(
         "admin_master_api_key_client",
     ]
 )
-def admin_client_new(request, admin_client_original, admin_master_api_key_client):
+def admin_client_new(
+    request: SubRequest,
```

Comment:
```suggestion
    request: pytest.FixtureRequest,
```

Reply:
When I check the type using pdb I get `SubRequest` as the type. Why should we switch it to `FixtureRequest`?

Reply:
1. `SubRequest` is a subtype of `FixtureRequest`.
2. `SubRequest` is not meant to be imported directly. `FixtureRequest` is a public export of pytest and is sufficient to be used as an annotation here.

Reply:
Ok you've convinced me. Will update.